portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,52 @@
+#info {
+  text-align:center;
+}
+
+#git{
+  text-align:center;
+}
+
+/*
+.row {
+  text-align:center;
+  display: flex;
+}
+
+.column {
+  text-align:center;
+  flex: 50%;
+  padding: 5px;
+}
+*/
```

Comment:
[nit] It's good practice to remove unused/uncomment code. Things like that tend to linger for years.